### PR TITLE
Move to grails 3.2.11/Pre 1.4.5 release

### PR DESCRIPTION
**Pull request by charvolant**
Update to grails 3.2.11. This is needed to overcome a baffling problem with the grails2 version where it would run but not maven install. Anyway. we should have done so a while back.
Update to JDK 1.8
JSON handling for sensitive instances is now explicit. This is because whatever magic grails2 was using no longer worked.

### CODE REVIEW

File: grails-app/i18n/messages.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-default.doesnt.match.message=Property [{0}] of class [{1}] with value [{2}] does not match the required pattern [{3}]
+1Gdefault.doesnt.match.message=Property [{0}] of class [{1}] with value [{2}] does not match the required pattern [{3}]
```
Comment (nickdos) [inline grails-app/i18n/messages.properties]: `1Gdefault ` looks like a typo - haven't checked subsequent commits so might already be fixed.

File: grails-app/init/sds/webapp2/BootStrap.groovy [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import au.org.ala.sds.model.ConservationInstance
+import au.org.ala.sds.model.PlantPestInstance
+import au.org.ala.sds.model.SensitivityCategory
+import au.org.ala.sds.model.SensitivityInstance
+import au.org.ala.sds.model.SensitivityZone
+import grails.converters.JSON
+
+class BootStrap {
+
+    def init = { servletContext ->
```
Comment (nickdos) [inline grails-app/init/sds/webapp2/BootStrap.groovy:15]: Would be nice to have some code comments here explaining why this is needed and what it does.

**Comment by nickdos** [comment]
I've looked over code and its all good to go ahead with merge (not sure if you want Dave to check as well). I did make 2 comments but they can be addressed later.

File: grails-app/i18n/messages.properties (hunk 1 above)
Comment (charvolant) [reply grails-app/i18n/messages.properties]: vi user detected. Fixed.

File: grails-app/init/sds/webapp2/BootStrap.groovy (hunk 2 above)
Comment (charvolant) [reply grails-app/init/sds/webapp2/BootStrap.groovy:15]: Commented

**Review by nickdos — APPROVED** [review]

**Pull request merged**